source/guide_owncloud.rst

```diff
@@ -0,0 +1,97 @@
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/owncloud.png 
+      :align: center
+
+#########
+ownCloud
+#########
+
+ownCloud_ is an open source cloud solution written in PHP and distributed unter the GAPLv3 licence.
+
+ownCloud was initially released in 2010 and is maintained by the ownCloud GmbH.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * MySQL_ 
+  * domains_
+  * cronjobs_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+If you want to use your cloud with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then download and the latest release of the ownCloud and extract it:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ curl https://download.owncloud.org/community/owncloud-42.23.1.tar.bz2 | tar -xjf - --strip-components=1
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  100 37.5M  100 37.5M    0     0  5274k      0  0:00:07  0:00:07 --:--:-- 6062k
+ [isabell@stardust html]$
+
+Now point your browser to your uberspace URL and follow the instructions.
+
+You will need to enter the following information:
+  
+  * admin username and password, here you can insert the credentials you want to use for the admin user
+
+If you want to use MySQL insert of SQLite you also need:
+
+  * your MySQL hostname, username and password: the hostname is ``localhost`` and you should know your MySQL credentials_ by now. If you don't, start reading again at the top.
+  * your ownCloud database name: we suggest you use an additional_ database. For example: isabell_owncloud
+
+Tuning
+======
+
+To get the best performance ownCloud suggests to add a local cronjob.
+
+Add the following cronjob to your cronjobs_ configuration:
+
+.. warning:: Replace ``<username>`` with your username!
+
+::
+
+ *  *  *  *  * /usr/bin/php71 -f var/www/virtual/<username>/html/cron.php
```

Comment:
The path to the script is missing a `/` in the beginning. Please don't link to the php binary directly, use our wrapper `/usr/bin/php`.